README.md

```diff
@@ -178,6 +178,14 @@ NOTE: You do not need to specify *end* segment - it will be added automatically.
 |`BULLETTRAIN_GO_FG`|`white`|Foreground color
 |`BULLETTRAIN_GO_PREFIX`|`go`|Prefix of the segment
 
+### AWS Profile
+
+|Variable|Default|Meaning
+|--------|-------|-------|
+|`BULLETTRAIN_AWS_BG`|`yellow`|Background color
+|`BULLETTRAIN_AWS_FG`|`black`|Foreground color
+|`BULLETTRAIN_AWS_PREFIX`|`☁️  `|Prefix of the segment
```

Comment:
Do we want to use spaces in 'prefix' variable? Wouldn't it be better to add space in prompt_aws when executing prompt_segment? (This is somehow connected to #209). Also, 2 spaces? on github it doesn't look like much but in console it will be much wider.